syntax.php

```diff
@@ -201,13 +201,20 @@ function render($mode, &$renderer, $data) {
       }
       return true;
     }
+    
+    if ($mode == 'odt') {
```

Comment:
I don't know about other rendering engines and their requirements, so I was quite defensive. On the other hand, what could go wrong? So the xhtml-export is the only export with the 'hidden' functionality. I'm ok with that.